components/ui/SyntaxHighlight.vue

```diff
@@ -1,12 +1,14 @@
 <template>
   <div class="syntax-highlight">
-    <button
+    <cv-button
+      size="small"
       title="Copy to clipboard"
+      class="syntax-highlight__copy-button"
       @click="copyToClipboard"
     >
       Copy
-    </button>
-    <pre v-highlightjs><code ref="code" :class="lang"><slot /></code></pre>
+    </cv-button>
+    <pre v-highlightjs="code"><code :class="`${lang} syntax-highlight__code`" /></pre>
```

Comment:
The [array syntax](https://vuejs.org/v2/guide/class-and-style.html#Array-Syntax) is more succint here and the preferred way when you need to mix dynamic and non-dynamic classes.

Reply:
Per offline conversation, I'm going to use `class` and `:class` as shown [here](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax)